src/components/two-factor-authentication/TwoFactorAuthenticationSetUp.vue

```diff
@@ -52,27 +86,40 @@
     <template v-slot:default="{ isActive }">
       <LayoutCard
         :closable="true"
-        :header="$t('admin.person.twoFactorAuthentication.setUpLong')"
+        :header="dialogHeader"
         @onCloseClicked="close2FADialog(isActive)"
+        data-testid="two-factor-authentication-dialog"
       >
-        <v-card-text>
+        <v-card-text v-if="!requestedSoftwareToken">
           <v-container>
             <v-row class="text-body bold px-md-16">
               <v-col>
                 <v-radio-group v-model="selectedOption">
                   <v-radio
                     :label="$t('admin.person.twoFactorAuthentication.softwareTokenOption')"
+                    data-testid="software-token-option"
```

Comment:
```suggestion
                    data-testid="software-token-radio-button"
```
Nicht notwendig, aber im Test später eindeutiger